packages/fxa-auth-server/lib/redis.js

```diff
@@ -4,37 +4,125 @@
 
 'use strict';
 
-const error = require('./error');
+const Redis = require('ioredis');
+const { readdirSync, readFileSync } = require('fs');
+const { basename, extname, resolve } = require('path');
 
-module.exports = (config, log) => {
-  const redis = require('../../fxa-shared/redis')(config, log);
-  if (!redis) {
-    return;
+const scriptNames = readdirSync(resolve(__dirname, 'luaScripts'), {
+  withFileTypes: true,
+})
+  .filter(de => de.isFile() && extname(de.name) === '.lua')
+  .map(de => basename(de.name, '.lua'));
```

Comment:
What does `de` stand for?  I can't figure it out.

Reply:
[Directory Entry](https://nodejs.org/dist/latest-v12.x/docs/api/fs.html#fs_class_fs_dirent) but if you need to ask it's prob a bad shorthand